diff --git a/node_modules/csrf-csrf/lib/cjs/index.cjs b/node_modules/csrf-csrf/lib/cjs/index.cjs
index 0ae0b17..e0b8e59 100644
--- a/node_modules/csrf-csrf/lib/cjs/index.cjs
+++ b/node_modules/csrf-csrf/lib/cjs/index.cjs
@@ -74,6 +74,8 @@ function doubleCsrf({ getSecret, getSessionIdentifier = () => "", cookieName = "
             const expectedHash = (0, crypto_1.createHash)("sha256")
                 .update(`${sessionIdentifier}${csrfToken}${secret}`)
                 .digest("hex");
+                console.log("Module expectedHash: ", expectedHash);
+                console.log("Module csrfTokenHash: ", csrfTokenHash);
             if (csrfTokenHash === expectedHash)
                 return true;
         }
@@ -89,6 +91,16 @@ function doubleCsrf({ getSecret, getSessionIdentifier = () => "", cookieName = "
         const possibleSecrets = Array.isArray(getSecretResult)
             ? getSecretResult
             : [getSecretResult];
+        console.log("Module csrfToken: ", csrfToken);
+        console.log("Module csrfTokenFromRequest: ", csrfTokenFromRequest);
+        console.log("Module csrf token match: ", csrfTokenFromRequest == csrfToken);
+        console.log("Module csrf token hash match: ", validateTokenAndHashPair({
+            csrfToken: csrfTokenFromRequest,
+            csrfTokenHash,
+            possibleSecrets,
+            sessionIdentifier: getSessionIdentifier(req),
+        }));
+
         return (csrfToken === csrfTokenFromRequest &&
             validateTokenAndHashPair({
                 csrfToken: csrfTokenFromRequest,
